recipes/n88util/bld.bat

```diff
@@ -0,0 +1,19 @@
+
+:: Create build directory
+mkdir build
+cd build
+set BUILD_CONFIG=Release
+
+:: CMake
+cmake .. ^
+	-G "Ninja" ^
+  -DCMAKE_BUILD_TYPE=%BUILD_CONFIG% ^
```

Comment:
indentation

Reply:
Thank you! I had spaces in some places, tabs in others. Very dumb.